README.md

```diff
@@ -0,0 +1,35 @@
+# Mouse Recorder
+###A cheap, modular, behavior recording program
+![](http://s16.postimg.org/yek83sdv9/Mouserecord_logo.jpg 2)
+
+# Purpose
+
+The program's is designed to record rodent behavior at specific intervals of time, based on the initiation of a trigger event (e.g., the press of a lever.) Moreover, the program operates through Raspberry Pi-based system to significantly reduce cost as opposed to conventional recording apparatus.
+
+# Building
+
+To install, use standard python installation procedure:
+    `python setup.py install`
+Additionally, the program requires installation of `setuptools`; the link to the installation instructions is as follows: <https://pypi.python.org/pypi/setuptools>
```

Comment:
Glad this is here. I think we can make this a bit shorter if we just make setuptools a link. Take a look at this for an example. ( https://github.com/jakirkham/docker_centos_drmaa_conda/blob/master/README.md#purpose )